### PR TITLE
3 login

### PR DESCRIPTION
**Pull request by Ashnayak**
Please review the login page.

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                <input name='username' type="email" placeholder="Enter your email" required>
+              </div>
+              <div class="input-box">
+                <i class="fas fa-lock"></i>
+                <input name='password' type="password" placeholder="Enter your password" required>
+              </div>
+              <div class="button input-box">
+                <input type="submit" value="Sumbit">
+              </div>
+              <div class="text sign-up-text">Don't have an account? <label for="flip">Sigup now</label></div>
```
Comment (surajdm123) [inline templates/login.html]: Signup is misspelled.

File: templates/login.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+              <div class="input-box">
+                <i class="fas fa-envelope"></i>
+                <input type="text" placeholder="Enter your email" required>
+              </div>
+              <div class="input-box">
+                <i class="fas fa-lock"></i>
+                <input type="password" placeholder="Enter your password" required>
+              </div>
+              <div class="button input-box">
+                <input type="submit" value="Sumbit">
```
Comment (surajdm123) [inline templates/login.html]: Submit is misspelled.

File: templates/login.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+              <div class="input-box">
+                <i class="fas fa-envelope"></i>
+                <input name='username' type="email" placeholder="Enter your email" required>
+              </div>
+              <div class="input-box">
+                <i class="fas fa-lock"></i>
+                <input name='password' type="password" placeholder="Enter your password" required>
+              </div>
+              <div class="button input-box">
+                <input type="submit" value="Sumbit">
```
Comment (surajdm123) [inline templates/login.html]: Submit is misspelled.

File: templates/login.html (hunk 1 above)
Comment (Ashnayak) [reply templates/login.html]: Good catch! Thanks, I'll correct this.

**Comment by Ashnayak** [comment]
Please Review @surajdm123

**Review by Himanshuu-Gupta — COMMENTED** [review]
Functionality is implemented properly. Can merge to main

**Comment by ShreyanshPrajapati** [comment]
LGTM

**Review by ShreyanshPrajapati — COMMENTED** [review]
Please include the modules required for running this file in the requirements.md file

**Pull request merged**